src/de/deserializer.zig

```diff
@@ -270,21 +270,19 @@ pub fn Deserializer(comptime dbt: anytype, comptime Reader: type) type {
                             return error.Overflow;
                         }
 
-                        break :blk try parseInt(Value, slice);
+                        return visitor.visitInt(ally, De, try parseInt(Value, slice));
```

Comment:
Missing a `try` here.

```suggestion
                        return try visitor.visitInt(ally, De, try parseInt(Value, slice));
```

Reply:
Is there a reason for using `return try`? It always seems redundant to me.

Reply:
AFAIK, there isn't any technical reason to use `return try`; it _is_ a redundant thing to do. Regardless, I personally like using it for readability purposes. Its nice being able to know which functions return error unions and which don't just by looking for a `try` (or `catch`). Plus it never hurts for maintenance purposes, say, if we decide to move the return statement further down the function or something.

In any case, I'll leave it up to you!

Reply:
Sure I'll add it if that's the projects preferred style